marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadHubTemplatesCommand.java

```diff
@@ -36,13 +36,13 @@ public void execute(CommandContext context) {
         try {
             JSONDocumentManager documentManager = hubConfig.newStagingClient(hubConfig.getStagingSchemasDbName()).newJSONDocumentManager();
             ResourcePatternResolver resolver = new PathMatchingResourcePatternResolver(getClass().getClassLoader());
-            for (Resource r : resolver.getResources("classpath*:/tdes/**/*.json")) {
-                File file = new File("tdes/" + r.getFilename());
+            for (Resource r : resolver.getResources("classpath*:/hub-templates/**/*.json")) {
```

Comment:
Taking a second look at this - this approach of finding files on the classpath makes sense for e.g. figuring out what databases/roles/etc to deploy. But once we've loaded the DHF modules into the modules database, it seems the simplest thing to do is have an e.g. "installJobTemplates" function in job-query-lib, which will write the template to both schema databases.

We then just need a single command to call a single DS endpoint to finish deployment of DHF after all the resources/modules have been deployed - e.g. "finishHubDeployment". The concept is that once we've got all the modules in place, we can invoke a DS endpoint that knows what else needs to be done. 

That also moves all of this logic from Java code into ML, which is typically what we want to do. 

Thus, I'm proposing the following:

- Change this command to "FinishHubDeploymentCommand"
- Make a new endpoint in the "data-services/system" directory called "finishHubDeployment"
- Have that endpoint call job-query-lib:installJobTemplates() (there's only one template, but there could clearly be a Job/Job template in the near future)

I know that's some rework now, but it'll make life easy for when we have future deployment needs that we can handle via DHF modules - we just invoke that code in the finishHubDeployment endpoint, and we don't have to touch any Java code.